coding-style.md

```diff
@@ -508,3 +508,33 @@ When integrating 3rd-party code into the code base, use the following scaffold:
 We use GitHub Actions to build and test each commit. Merging a pull request
 requires that all checks pass for the latest commit in the branch. GitHub
 displays the status of the individual checks in the pull request.
+
+### Code Coverage
+
+The GitHub Actions workflow [Analysis][analysis-workflow] contains a _Code
+Coverage_ job that runs unit tests for libvast and bundled plugins, and
+integration tests for VAST and VAST with bundled plugins to create a detailed
+line coverage report. The CI creates and uploads reports as an artifact in the
+Analysis workflow as part of every pull request and for every merge to master.
+
+Note that a branch being covered does not imply that the tests covering it are
+semantically correct, but the report is still is a valuable indicator
+especially for new tests to see whether a branch is not being tested at all.
```

Comment:
```suggestion
```

I think we must assume that people understand what code/test coverage is. Explaining it here feels a bit odd.